docs/Paymaster/paymastermethods.md

```diff
@@ -5,39 +5,119 @@ sidebar_position: 3
 
 # Paymaster Methods
 
-This section will cover methods that can be called using the Paymaster Package.
-
-:::note
-
-When using these methods you will need to create a `userOp`. The [accounts methods](/account/methods) will help you in creating these for the paymaster methods below.
-
-:::
-
 Imports needed for these methods:
 
 ```ts
-import {
-  IHybridPaymaster,
-  PaymasterFeeQuote,
-  PaymasterMode,
-  SponsorUserOperationDto,
-} from "@biconomy/paymaster";
-```
-
-After setting up your smart account as mentioned above you can start using the Paymaster in this way:
+import { IHybridPaymaster, PaymasterFeeQuote,
+ PaymasterMode, SponsorUserOperationDto} from "@biconomy/paymaster";
 
-```ts
 const biconomyPaymaster =
   smartAccount.paymaster as IHybridPaymaster<SponsorUserOperationDto>;
 ```
 
```

Comment:
Would it make sense to split this section into two parts: `SPONSORED` and `ERC20`
Somone who knows which flow to use can just focus on that